beacon-chain/db/filesystem/metrics.go

```diff
@@ -0,0 +1,27 @@
+package filesystem
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	blobSaveLatency = promauto.NewHistogram(prometheus.HistogramOpts{
+		Name:    "blob_storage_save_latency_seconds",
+		Help:    "Latency of blob storage save operations",
+		Buckets: prometheus.DefBuckets,
+	})
+	blobFetchLatency = promauto.NewHistogram(prometheus.HistogramOpts{
+		Name:    "blob_storage_get_latency_seconds",
+		Help:    "Latency of blob storage get operations",
+		Buckets: prometheus.DefBuckets,
+	})
+	blobsPrunedCounter = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "blob_pruned_blobs_total",
+		Help: "Total number of pruned blobs.",
+	})
+	blobsTotalGauge = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "blob_total_number",
```

Comment:
At start up, you need to set this value with `blobsTotalGauge.Set`.

I think the best way to do that is to count all of the remaining blobs after a prune, since you run prune at start up anyway.